packages/mdc-tabs/README.md

```diff
@@ -0,0 +1,325 @@
+# MDC Tabs
```

Comment:
I'm going to hold off on doing too much of a detailed review of this README until I do a first pass on a component, but some general comments:

- The usage section is missing a _lot_ of information regarding how best to use tabs. This is the primary location which users will be reading in order to figure out how to best accomplish their desired use case with our component. Given the breadth and complexity of tabs, this needs to be filled out much more. A good strategy would be to ask yourself if you wanted to accomplish a task using MDCTabs, or find out if it supports something specific, could you find it within the usage section? The [mdc-list usage section](https://github.com/material-components/material-components-web/tree/master/packages/mdc-list#usage) is a pretty good starting point for what this should eventually look like.
- The import paths for the `Including in code` section are incorrect, and there's no section that talks about including the Sass. 
- The information architecture regarding the component / foundation / adapter APIs for each component could be a bit more clear. You may want to consider grouping them under the `MDCTab`, `MDCTabs`, and `MDCTabsScroller` sections respectively.